cr_electronic_invoice/models/account_invoice.py

```diff
@@ -501,17 +501,30 @@ def load_xml_data(self):
         product = False
         load_lines = bool(self.env['ir.config_parameter'].sudo().get_param('load_lines'))
```

Comment:
I think this parameter would be better after purchase_journal, in order to get the information from the purchase journal 

load_lines = purchase_journal.load_lines

@EstebanMonge